namespace/type-prefix-registry.md

```diff
@@ -0,0 +1,10 @@
+| Type Prefix | Class Name | Definition | Example | Reference |
+|-------------|------------|------------|---------|-----------|
+| SQ | Sequence | ... | `ga4gh:SQ:...` | [VRS Computed Identifers](https://vrs.ga4gh.org/en/1.2.0/impl-guide/computed_identifiers.html)|
+| VA | Allele | ... | `ga4gh:VA.EgHPXXhULTwoP4-ACfs-YCXaeUQJBjH_` | [VRS Computed Identifers](https://vrs.ga4gh.org/en/1.2.0/impl-guide/computed_identifiers.html) |
+| VH | Haplotype | ... | `ga4gh:VH:...` | [VRS Computed Identifers](https://vrs.ga4gh.org/en/1.2.0/impl-guide/computed_identifiers.html) |
+| VAB | Abundance | ... | `ga4gh:VAB:...` | [VRS Computed Identifers](https://vrs.ga4gh.org/en/1.2.0/impl-guide/computed_identifiers.html) |
```

Comment:
I'm pretty sure we changed Abundance to CopyNumber and I think VAB should be VCN (Alex W can verify)

Reply:
@larrybabb is correct. Though in fairness we haven't updated the documentation on VRS latest to reflect this. Made an issue to address this [here](https://github.com/ga4gh/vrs/issues/341).

Reply:
Should I wait until the VRS PR merges before I update it here?

Reply:
Yes, please wait for 1.2.1; I will put in PRs this week so that we may merge and release soon.